mpc_controller:
  robot_name: "mobile_manipulator"
  base_link: $(arg prefix)base_link
  tool_link: $(arg prefix)tool_frame
  simulation: $(arg simulation)
  joint_names:
    - $(arg prefix)joint_1
    - $(arg prefix)joint_2
    - $(arg prefix)joint_3
    - $(arg prefix)joint_4
    - $(arg prefix)joint_5
    - $(arg prefix)joint_6
    - $(arg prefix)joint_7
  mpc_frequency: 20
  command_topic: "/joints_position_controller/command"
  path_topic: "/desired_path"

  admittance: true
  wrench_topic: "/ft_compensated"

  kp_linear_gains: [0.01, 0.01, 0.01]
  kp_angular_gains: [0.001, 0.001, 0.001]
  ki_linear_gains: [ 0.0, 0.0, 0.0 ]
  ki_angular_gains: [ 0.0, 0.0, 0.0 ]
  force_integral_max: [1.0, 1.0, 1.0]
  torque_integral_max: [1.0, 1.0, 1.0]

  force_threshold: [0.0, 0.0, 0.0]
  torque_threshold: [0.0, 0.0, 0.0]